Tests/SwiftSyntaxMacroExpansionTest/MacroReplacementTests.swift

```diff
@@ -113,4 +114,230 @@ final class MacroReplacementTests: XCTestCase {
       """
     )
   }
+
+  func testMacroGenericArgumentExpansion_base() throws {
```

Comment:
We generally don’t use `_` in test names. I would prefer to name this `testMacroGenericArgumentExpansionBase`. Same for the other tests.

Reply:
Ok I'll follow the repo style on it, thanks